packages/web-app-files/src/components/SideBar/SideBar.vue

```diff
@@ -254,8 +255,11 @@ export default defineComponent({
     }
 
     watch(
-      selectedFiles,
+      () => [...unref(selectedFiles), props.open],
       (newResource, oldResource) => {
```

Comment:
The callback args `newResource` and `oldResource` are not a list of resources anymore, because you added an additional attribute in the watcher. They are now arrays of resources and an additional boolean. That breaks the check in line 275 on `!!oldResource`.

Reply:
Good catch, thx! I removed it completely, as we don't need it anymore. There is still room for some improvements when navigating with an opened sidebar, but it's already 100 times better than before.